lisa/tools/lspci.py

```diff
@@ -115,3 +115,20 @@ def enable_devices(self) -> None:
         self.node.tools[Echo].write_to_file(
             "1", self.node.get_pure_path("/sys/bus/pci/rescan"), sudo=True
         )
+
+    def get_used_module(self, slot: str, force_run: bool = False) -> str:
+        if force_run:
```

Comment:
This check is not necessary. The force_run is handled in the `run` method already.

Reply:
update